packages/shared/styles/components/SfBottomNavigation.scss

```diff
@@ -13,15 +18,39 @@
   padding: 0 1rem;
   background: white;
   box-shadow: 0px -2px 10px rgba(154, 154, 154, 0.15);
-  &__item > .sf-icon {
-    --icon-color: #{$c-gray-variant};
-  }
-  &__item--active > .sf-icon {
-    --icon-color: #{$c-black};
-  }
-  &__floating-icon {
+  &__item {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
     position: relative;
+    font-size: 12px;
+    color: $c-gray;
+
+    & > .sf-icon {
+      --icon-color: #{$bottom-navigation__item-color};
+    }
+
+    &--active {
+      color: $c-black;
+      font-weight: $body-font-weight-secondary;
+      & > .sf-icon {
+        --icon-color: #{$bottom-navigation__item-color--active};
+      }
+    }
+
+    &--has-margin {
+      margin-top: $spacer;
+    }
+  }
+
+  &__floating-button {
```

Comment:
please add `box-shadow: 0px 4px 6px rgba(66, 66, 66, 0.2);` for --floatink like on our design